CHANGELOG

```diff
@@ -1,4 +1,5 @@
 [master]
+    - Removes button to merge entries and keep the old ones.
```

Comment:
I suggest to add the context: "Merge Entries Dialog:" and "Use only one button to perform merge" might be a better description?